e2e/pvc.go

```diff
@@ -75,6 +75,21 @@ func createPVCAndvalidatePV(c kubernetes.Interface, pvc *v1.PersistentVolumeClai
 		}
 
 		if pvc.Spec.VolumeName == "" {
+			var events *v1.EventList
+			// Log the events for the PVC if its not bound yet
+			events, err = c.CoreV1().Events(namespace).List(ctx, metav1.ListOptions{
+				FieldSelector: "involvedObject.kind=PersistentVolumeClaim,involvedObject.name=" + name,
+			})
+			if err != nil {
+				framework.Logf("error getting events for PVC %s: %v", name, err)
+
+				return false, nil //nolint:nilerr // intentionally returning nil on error.
```

Comment:
nit: you could structure this differently and not have the `//nolint` annotation when the for-loop is inside `if err == nil { ... }`.